low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -87,6 +95,16 @@ def get_health_info(self, rpath):
         fru_found = False
         nodes = rpath.strip().split(">")
         leaf_node, _ = self.get_node_details(nodes[-1])
+        try:
+            info["name"] = BuildInfo().get_attribute("NAME")
+            info["version"] = BuildInfo().get_attribute("VERSION")
+            info["build"] = BuildInfo().get_attribute("BUILD")
+        except BuildInfoError as err:
+            logger.error(self.log.svc_log(
+                f"Unable to get build info due to {err}"))
+        except Exception as err:
```

Comment:
both except blocks do the same handling. You can simply use except (BuildInfoError, Exception)

Reply:
done

Reply:
I can still see two except blocks

Reply:
@jjxsg avoid generic exception handling here

Reply:
Sorry, I missed this comment and though I have made the necessary changes. 
Actually, I was following what was done for SAS and Network ports, thus added generic Exceptions. 
I have removed the generic exceptions.